google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml

```diff
@@ -0,0 +1,162 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.devtools.clouddebugger.spi.v2
+  python:
+    package_name: google.cloud.gapic.devtools.clouddebugger.v2
```

Comment:
I think the issue with changing proto paths/directories is that there are potentially other consumers outside of the vkit toolchain that depend on the protos. But, one thing to consider is that this was only just recently released, so potentially the impact would be minimal to change it; the flip side is there might be internal consumers who depend on the current structure. I think the larger issue is that we just don't know the impact and we don't have a policy around it - we aren't following something like semantic versioning for protos. 

Absent changing that, I think we just have to define the api short name ourselves.